Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 The OPA Authors.  All rights reserved.
+# Use of this source code is governed by an Apache2
+# license that can be found in the LICENSE file.
+
+# global ARG vars, see https://github.com/moby/moby/issues/37345
+ARG GOVERSION=1.12.9
```

Comment:
Will this become the only place where we need to define this (i.e., will it be removed from the makefile and travis yaml)? /me prepares confetti